evap/context_processors.py

```diff
@@ -8,3 +8,7 @@ def slogan(request):
     if get_language() == "de":
         return {'SLOGAN': random.choice(settings.SLOGANS_DE)}  # nosec
     return {'SLOGAN': random.choice(settings.SLOGANS_EN)}  # nosec
+
+
+def debug(request):
+    return {'DEBUG': settings.DEBUG}
```

Comment:
If we lowercased `SLOGAN`, we could also lowercase `DEBUG` :)
Also, would it be possible to use this context processor in the nav bar for the striped background?

Reply:
it is possible to use the value from the context processor but we would have to forward it in the navbar inclusion, so i think there is not really an advantage to the current solution.